### DISCUSSION TITLE
YT Revanced v7a version?

### DISCUSSION THREAD

**Discussion by NAEL-56** [ReVanced/revanced-discussions#1127] [Question] [answered]
Sorry, it's my first time here. Isn't there a v7a version of YT Revanced? The normal one isn't installing. Since v7a was the only YT Music version my Redmi 9A could install, I was looking for something similar for the standard YT Revanced. I suspect my phone runs on a 32bit architecture, but I couldn't confirm it anywhere. Thank you for your work ^^

**Comment by validcube** [comment, marked as answer]
👋 Hello, @Irogrig!

We do support `armeabi-v7a` architecture, please try it out by grabbing the APK file for YouTube compiled for `armeabi-v7a` architecture (if you're unsure of which architecture should be use, try `universal`) and patch them using ReVanced Manager or ReVanced CLI.

**Reply by NAEL-56** [reply]
Hi, I tried downloading from the site and from the Revanced Manager, but it wouldn't install neither version. The website does have older versions for download, but neither said v7a compatible. Is there a specific link with the version I have to download I'm missing? Revanced Manager also doesn't seem to have anything except the basic newest version download function. Again, sorry, total noob here 😅

**Reply by Ushie** [reply]
Join the discord server for support please https://revanced.app/discord

**Reply by LisoUseInAIKyrios** [reply]
And make sure you are using the website ReVanced.app

There are numerous imposter ReVanced sites distributing bogus software.

**Reply by NAEL-56** [reply]
@LisoUseInAIKyrios wow, I was using Revanced.net! Dangerous

**Reply by NAEL-56** [reply]
Thanks guys, was able to solve it with your help.
